src/bicep/mlz.bicep

```diff
@@ -398,7 +398,7 @@ param hubSubscriptionId string = subscription().subscriptionId
 param identitySubscriptionId string = hubSubscriptionId
 param operationsSubscriptionId string = hubSubscriptionId
 param sharedServicesSubscriptionId string = hubSubscriptionId
-
+param deploymentType string = 'MissionLandingZoneARM'
```

Comment:
This follows the pattern in place of exposing most of the settings as parameters, but in this case I think this may be better as a hard coded value in the tags object instead of as a parameter. Two reasons for this: the name `deploymentType` may be confused by people as something that affects the deployment (other than as a tag), and the purpose of this tag is to identify the objects created by an MLZ deployment so having it as a parameter would indicate that it's an intentional customization point, which it is not. However, I'm very open to being wrong on all counts here. 😃

Reply:
Interesting, I attempted to leave it as a hard-coded object in the tags object but it failed to run with the following message: 
`Error BCP057: The name "deploymentType" does not exist in the current context.`
I had to add this line in order to get it to run without issue. 
I am open to suggestions on how to get this to work in a different way if that is the preferred outcome.